README.md

```diff
@@ -1,2 +1,213 @@
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Car APP - Back End <a name="about-project"></a>
+
+** Car APP - Back End ** Is a Ruby Back-End project for users to rent cars, and Admins to create and delete cars.  Users will have the chance to view the history of cars reserved.
+
+** The app front end can be found [here](https://github.com/gjuliao/car_app_frontend)
+
+## 🛠 Built With <a name="built-with"></a>
+
+
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/documentation/">Ruby on Rails</a></li>
+  </ul>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Users can view Cars signup, login and reserve cars.**
+- **Admin can login, add and delete cars.**
+- **Users can view history of cars**
+- **Users can view characteristics of cars (price, electric, brand and much more).**
+- **Users can view which cars are available for rent and which aren't**
+- **Full stack app**
+- **Styling uses CSS**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby](https://www.ruby-lang.org/en/) installed and running
+- [PostgreSQL](https://www.postgresql.org/) installed
+- [RubyonRails](https://rubyonrails.org/) installed
+
+### Setup
+
+Clone this repository to your desired folder:
+
+``` bash
+git clone git@github.com:dgcuenca/car_app_backend.git
+```
+and then move to the folder: 
+
+```bash
+cd car_app_backend
+```
+
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Install dependancies with:
+
+```bash
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Create a master.key file in the config folder and paste  the key that was provided
+```
+code ./config/master.key
+```
+
+```console
+bundle exec rails db:create
+bundle exec rails db:migrate
+bundle exec rails db:seed
+bundle exec rails start
+```
+then go to [http://localhost:300](http://localhost:300)
+
+Users to log in into the app:
+```
+admin: admin@admin.com
+password: 123456
+
+user: diego@gmail.com
+password: 123456
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+rspec spec/models/ // to test models
+rspec spec/requests/api/v1  // to test controllers
+```
```

Comment:
- [ ] Some tests are not passing on my end.  Kindly consider fixing them so they all pass:

Models:

![image](https://github.com/gjuliao/car_app_backend/assets/57421823/dc0287ca-0513-4744-a1f9-755fff7f07ce)

models/user_spec.rb is not yet implemented.  Kindly either implement it or remove it.

Requests:

![image](https://github.com/gjuliao/car_app_backend/assets/57421823/b9975f8e-1df5-4f48-ae0e-e6369f8195bb)

requests/api/v1/user_spec.rb is not yet implemented.  Kindly either implement it or remove it.